OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh add edit button to each summary

### Description

![BUG DESCRIPTION](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/d9738406-556e-494f-a9dc-9574e439c792)


### Related PRS (if any):


### Main changes explained:

    Update file Timelog/WeeklySummaries.jsx. Added the functionalities of editing an already submitted summary
     Update file Timelog.css to add style to the buttons


### How to test:

    check into current branch
    do npm run start:local to run this PR locally
    log as owner/admin/core/mentor/manager/volunteer user
    accounts that can submit summary for other users (owner/admin/manager/mentor) can also edit summary for them  hence when switched to different user profile they could access the edit button. Core/Volunteer account can only edit their own summary.
    edited summary will only be saved if it has more than 50 words.       

**Added Functionalities**

User Able to edit its own Summary

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/7132a057-711b-46ad-80b3-256106b38d23

Admin User able to edit summary for different User 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/83bd474e-a0bf-4c2c-bd19-797dd679e3f0
